### PR TITLE
Update virt (KubeVirt virtctl plugin package) to v0.30.0

### PR DESCRIPTION
**Pull request by dhiller**
See https://github.com/kubevirt/kubevirt/releases/tag/v0.30.0

### CODE REVIEW

**Comment by ahmetb** [comment]
:robot: _Beep beep! I’m a robot speaking on behalf of @ahmetb._ :robot:

-----

This pull request **does not** seem to be a straightforward version bump. I'll have a human review this.

_Why wasn't this detected as a plugin version bump:_

>file plugins/virt.yaml is not a straightforward version bump: diff line 3 unrecognized for version bumps: [-  description: |]

**Comment by dhiller** [comment]
Hey! Just some minor changes to caveat and description. PTAL, thanks!

**Comment by dhiller** [comment]
Hey @ahmetb @corneliusweig PTAL, thanks!

File: plugins/virt.yaml [hunk 1]
```diff
@@ -58,17 +58,23 @@ spec:
...
+
+    for a usage example
+  description: |
+    virt plugin is a wrapper for virtctl originating from the KubeVirt project. KubeVirt is a virtualization add-on to
+    Kubernetes, i.e. it enables to run existing virtual machines on Kubernetes clusters. virtctl controls virtual
+    machine related operations on your Kubernetes cluster like connecting to the serial and VNC consoles.
+
     Kubevirt documentation:
       Overview:
         https://kubevirt.io/
```
Comment (corneliusweig) [inline plugins/virt.yaml:80]: The link to this page will automatically be shown for `krew info`, so it should not be shown twice.

File: plugins/virt.yaml [hunk 2]
```diff
@@ -58,17 +58,23 @@ spec:
...
     virt plugin is a wrapper for virtctl originating from the KubeVirt project. In order to use virtctl you will
     need to have KubeVirt installed on your Kubernetes cluster to use it. See https://kubevirt.io/ for details
-  description: |
-    virt plugin is a wrapper for virtctl originating from the KubeVirt project. KubeVirt is a virtualization add-on to
-    Kubernetes, i.e. it enables to run existing virtual machines on Kubernetes clusters. virtctl controls virtual
-    machine related operations on your Kubernetes cluster like connecting to the serial and VNC consoles.
 
     Run
 
       kubectl virt help
```
Comment (corneliusweig) [inline plugins/virt.yaml:64]: What will happen if you run `kubectl virt`, will it also show the help Text? If so, consider removing this, to keep it as short as possible.

Comment (dhiller) [reply plugins/virt.yaml:64]: Actual output of latest v0.30.0 version looks like this:
```
$ kubectl virt version
Client Version: version.Info{GitVersion:"v0.30.0-49+d358cf085b5a86", GitCommit:"d358cf085b5a86cc4fa516215f8b757a4e61def2", GitTreeState:"clean", BuildDate:"2020-06-09T12:22:50Z", GoVersion:"go1.12.8", Compiler:"gc", Platform:"linux/amd64"}
the server could not find the requested resource
$ kubectl virt
Available Commands:
  console      Connect to a console of a virtual machine instance.
  expose       Expose a virtual machine instance, virtual machine, or virtual machine instance replica set as a new service.
  help         Help about any command
  image-upload Upload a VM image to a DataVolume/PersistentVolumeClaim.
  migrate      Migrate a virtual machine.
  pause        Pause a virtual machine
  rename       Rename a stopped virtual machine.
  restart      Restart a virtual machine.
  start        Start a virtual machine.
  stop         Stop a virtual machine.
  unpause      Unpause a virtual machine
  version      Print the client and server version information.
  vnc          Open a vnc connection to a virtual machine instance.

Use "kubectl virt <command> --help" for more information about a given command.
Use "kubectl virt options" for a list of global command-line options (applies to all commands).
```

File: plugins/virt.yaml (hunk 1 above)
Comment (dhiller) [reply plugins/virt.yaml:80]: OK, will remove it.

**Comment by ahmetb** [comment]
:robot: _Beep beep! I’m a robot speaking on behalf of @ahmetb._ :robot:

-----

This pull request **does not** seem to be a straightforward version bump. I'll have a human review this.

_Why wasn't this detected as a plugin version bump:_

>file plugins/virt.yaml is not a straightforward version bump: diff line 3 unrecognized for version bumps: [+]

**Comment by dhiller** [comment]
@corneliusweig PTAL, thanks!

**Pull request merged**